nifi-commons/nifi-calcite-utils/src/main/java/org/apache/nifi/sql/internal/NiFiProjectTableScanRule.java

```diff
@@ -68,8 +73,7 @@ private static int[] getProjectionFields(final List<RexNode> expressions) {
     public interface Config extends RelRule.Config {
         // This impl comes directly from the Calcite documentation.
```

Comment:
With the recent changes, is this comment still relevant i.e. is this code still directly from the Calcite documentation?

Reply:
I guess this was coming from https://calcite.apache.org/javadocAggregate/org/apache/calcite/plan/RelRule.html
So yeah we can probably remove that line.

```suggestion
```